server/svix-server/src/core/permissions.rs

```diff
@@ -157,3 +157,36 @@ impl FromRequestParts<AppState> for ApplicationWithMetadata {
 struct ApplicationPathParams {
     app_id: ApplicationIdOrUid,
 }
+
+pub enum EventTypeFeatureFlagPermissions {
+    Organization,
+    Application(EventTypeFeatureFlagSet),
+}
+
+#[async_trait]
+impl FromRequestParts<AppState> for EventTypeFeatureFlagPermissions {
+    type Rejection = Error;
```

Comment:
Just to add to the above - it seems fine to me if instead of a new type,  just add the `EventTypeFeatureFlagSet` directly to the `ReadAll` struct (or whichever `permissions::` struct needs it). The endpoints that don't need to read the flags can just ignore with `..`, which I think we do a lot of already.

Reply:
Yep, did exactly that, I'll resolve this thread so we can keep this topic in one convo.